chaste_codegen/chaste_model.py

```diff
@@ -309,8 +313,13 @@ def _get_cytosolic_calcium_concentration_var(self):
             return cytosolic_calcium_concentration
 
     def _get_modifiable_parameters_annotated(self):
-        """ Get the variables annotated in the model as modifiable parametery"""
-        return self._model.get_variables_by_rdf((self._PYCMLMETA, 'modifiable-parameter'), 'yes')
+        """ Get the variables annotated in the model as modifiable parameter
+
+            Also include V and cytosolic_calcium_concentration_var if these are constant and not stae_vars
+        """
+        return self._model.get_variables_by_rdf((self._PYCMLMETA, 'modifiable-parameter'), 'yes') +\
+            [p for p in (self._membrane_voltage_var, self._cytosolic_calcium_concentration_var)
```

Comment:
Better would be to set the modifiable-parameter annotation for these variables in earlier processing.

Reply:
ok